packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -833,412 +845,468 @@ describe('Select', () => {
         let disconnect: () => Promise<void>;
         let pageObject: SelectPageObject;
 
-        beforeEach(async () => {
-            ({ element, connect, disconnect } = await setup());
-            element.style.width = '200px';
-            element.filterMode = FilterMode.standard;
-            await connect();
-            pageObject = new SelectPageObject(element);
-        });
+        describe('with filterMode = standard', () => {
```

Comment:
All changes from here until line 1271 are indentation. Differ is a little confused in places.